website/pages/en/index.js

```diff
@@ -162,7 +162,7 @@ class HomeSplash extends React.Component {
           <div className="index-projects-boxes grid-blocks two-blocks-grid">
             <a href="https://cowswap.exchange" className="white-box">
                <strong>CowSwap</strong> <br></br>
-               CowSwap is a <strong> the first trading UI </strong> built on Gnosis Protocol v2 .
+               CowSwap is <strong> the first trading UI </strong> built on Gnosis Protocol v2 .
```

Comment:
they were already there, but I think we don't need those extra-spaces:
```suggestion
               CowSwap is <strong>the first trading UI</strong> built on Gnosis Protocol v2.
```

Although it looks like they are all over. Maybe @avsavsavs u can review that in general.

Reply:
Happy to remove all of those as well if that's desired! Just wasn't sure if it was intentional to give some additional spacing to the emphasized text. Let me know 😄